### ISSUE TITLE
Cirrus Analysis milestone CFR

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#3301]
The Cirrus analysis milestone 1 - work is tracked in this [project board](https://github.com/mozilla/jetstream/projects/3).

We are hoping these get into milestone 1 - but as they have dependencies on things landing in same time frame: putting in separate card.  There is value to landing these ASAP, as current x-man/CFR experiments can start leveraging and giving feedback:

* Stretch: Experiment intake tools may specify feature metrics that Jetstream will analyze.  TBD on the feature work being complete.
* Stretch: CFR specific metrics will be available in analysis.  Though CFR is not in the Nimbus A/A scope - we want to see CFR for the data analysis sooner.  CFR experiments can utilize the automated analysis and provide needed user comprehension/need feedback, before Nimbus intake is ready.  Automating CFR analysis is blocked on 1649642.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-21)
┆Issue Number: EXP-21
